test/powershell/Language/Scripting/MacLineEndings.Tests.ps1

```diff
@@ -0,0 +1,69 @@
+﻿Describe 'Tests for mac line endings' -Tags "CI" {
+    BeforeAll {        
+        $newline = "`n"

```

Comment:
Instead of using the same line ending for all tests, I'd put the line ending in `$testData` and use:

CR
LF
CR + LF

CR isn't too interesting (Mac OS up to 9), but the parser handles it fine, so it's good to test.  And maybe rename the file/`Describe` to just `line endings`.